### ISSUE TITLE
Unable to download image URL 

### ISSUE THREAD

**Issue by SamboVisal** [SDWebImage/SDWebImage#2538]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           |  ios 
 Platform Version        |  11.0 
 SDWebImage Version      | 4.4.2
 Integration Method      | cocoapods 
 Xcode Version           | Xcode 9
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | My own project
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Thank you for great tool! 
Now i am trying to fetch data from `Firestore` and load it to my app.
When load completed from server I can say `URL` of image is openable on the web but i don't know why it cannot download in my app. 
The error is `Error Domain=SDWebImageErrorDomain Code=0 "Downloaded image has 0 pixels" UserInfo={NSLocalizedDescription=Downloaded image has 0 pixels`.

I have been checking another issues but it cannot help me. So, i have to ask my own questions...

This is one link of my all images
[This is link](https://firebasestorage.googleapis.com/v0/b/vkclub-c861b.appspot.com/o/explore%2Faccommodation%2FAccommodation3.webp?alt=media&token=54a7f1c5-f601-44a1-9416-d1672bd85f5f)

This is how I fetching data to collectionView
![screen shot 2561-11-19 at 10 23 53 pm](https://user-images.githubusercontent.com/29918870/48716396-d8de1180-ec49-11e8-8f51-5978b8ad7131.png)

This is how I populate data to each cell
![screen shot 2561-11-19 at 10 25 43 pm](https://user-images.githubusercontent.com/29918870/48716502-180c6280-ec4a-11e8-81de-8733fe2b9703.png)

This is Error that i got in console

![screen shot 2561-11-19 at 10 26 53 pm](https://user-images.githubusercontent.com/29918870/48716592-44c07a00-ec4a-11e8-80b7-7d2c2763fd13.png)

Thank for your help!!!

**Comment by kinarob** [comment]
Hello, I run the image url [https://firebasestorage.googleapis.com/v0/b/vkclub-c861b.appspot.com/o/explore%2Faccommodation%2FAccommodation3.webp?alt=media&token=54a7f1c5-f601-44a1-9416-d1672bd85f5f](https://firebasestorage.googleapis.com/v0/b/vkclub-c861b.appspot.com/o/explore%2Faccommodation%2FAccommodation3.webp?alt=media&token=54a7f1c5-f601-44a1-9416-d1672bd85f5f) in our SDWebImage Demo project , and it can be displayed normally. Since the image url you requested is webp format, I would like to know if you have imported  `SDWebImage/WebP`？
